easyartshow/src/components/ArtBoard.js

```diff
@@ -82,32 +68,52 @@ function ArtBoard({ id }) {
   const roomRef = dbRef(db, "easyartshow/rooms/");
   const [captionList, setCaptionList] = useState([]);
 
-  function formatDate(dateString) {
-    const date = new Date(dateString);
-    const month = date.getMonth() + 1;
-    const day = date.getDate();
-    const year = date.getFullYear();
-    return `${month}/${day}/${year}`;
-  }
-
   const unsub = onSnapshot(doc(dbFireStore, "rooms", `${id}`), (doc) => {
```

Comment:
On `deletePhoto`, this does not update so the mapping part in the end still tries to pull the image that's deleted.